keyboards/bastardkb/charybdis/readme.md

```diff
@@ -18,32 +18,63 @@ Check out the [keyboard build guides](https://docs.bastardkb.com) for the Charyb
 
 ## Building the firmware
 
+You must specify the shield version when compiling/flashing the firmware. 
+
 ### Charybdis (4x6)
 
 The `default` keymap is inspired from the original [Dactyl Manuform](../../handwired/dactyl_manuform) layout.
 
+* For Elite-C v1 shield
+
+```shell
+	qmk compile -kb bastardkb/charybdis/4x6/v1/elitec -km default
+```
+* For Elite-C v2 shield
+
 ```shell
-qmk compile -kb bastardkb/charybdis/4x6 -km default
+	qmk compile -kb bastardkb/charybdis/4x6/v2/elitec -km default
 ```
 
 Check out the `via` layout if you're looking for VIA support:
 
+* For Elite-C v1 shield
+
+```shell
```

Comment:
I think the entire block needs to be indented by 4 spaces, not just the content of the code-block.  In the preview, the code lock should start at the same position as the first letter of the bullet point list.

Reply:
I switched it to be a table. I think that's clearer